src/components/Sidebar.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Sidebar() {
```

Comment:
Named this component as `Sidebar` instead of Popular Tags as i believe that there will be additional items here later on. If more items do get added in the future, I will create a `PopularTag` component separately.

Reply:
Also too generic of a name because there might be other types of sidebars that you would like. why don't you name it `TagsSideBar`, `TagsSelector ` or something similar.